tests/test_cfg_end_to_end_tests.py

```diff
@@ -175,99 +84,72 @@ def get_test_cases():
         (f"{DATASET_CONFIGS_ROOT}/kazakh/ksc2/config.yaml", data_check_fn_ksc2),
     ]
 
-
 def check_e2e_test_data() -> bool:
-    """Checks if required environment variables are defined for e2e data.
-
-    Either TEST_DATA_ROOT needs to be defined or both AWS_SECRET_KEY
-    and AWS_ACCESS_KEY.
-    """
```

Comment:
Please do not delete the docstring

Reply:
Just trying to fix ci/cd, this will be reverted, no worries